### DISCUSSION TITLE
Change default path color or room color

### DISCUSSION THREAD

**Discussion by Masteroshi430** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#165] [Q&A] [answered]
I have a problem where the vacuum path color and a room color are strictly the same (light blue):  
![rooms](https://user-images.githubusercontent.com/13667502/142144782-71d19f0b-159e-4b0a-94e2-c4a71c55c282.png)
The easier would be to change the default path color or the color of that room from my theme but I'm not sure if they are in that list:
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/css_variables.md

**Comment by PiotrMachowski** [comment, marked as answer]
This is something that has to be done in [camera's configuration](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#colors-configuration), not in card. Card just gets an image from the camera and can't modify it. Camera is not aware of current theme, so it can't use it

**Reply by Masteroshi430** [reply]
It works perfectly, thanks for your huge work on both!
